static/css/style.css

```diff
@@ -34,6 +34,17 @@ body {
 		padding-right: 20px;
 	}
 }
+.CClogo{
+	height: 40px;
+	width: 40px;
+	position: relative;
+	margin: 0px;
+	margin-right: 30px;
+	margin-left: 10px;
```

Comment:
Hello,

Please, could u remove me from this thread? I'm not collaborating in this
project.

Thanks,
Ana
El 12/1/2015 19:19, "Jure Cuhalev" <notifications@github.com> escribió:

> In static/css/style.css
> <https://github.com/CodeCatz/TrackCat/pull/74#discussion-diff-22813980>:
>
> > @@ -34,6 +34,17 @@ body {
> >  		padding-right: 20px;
> >  	}
> >  }
> > +.CClogo{
> > +	height: 40px;
> > +	width: 40px;
> > +	position: relative;
> > +	margin: 0px;
> > +	margin-right: 30px;
> > +	margin-left: 10px;
>
> This could be better written as:
>
> margin: 0 30px 0 10px;
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CodeCatz/TrackCat/pull/74/files#r22813980>.
>